modules/core/jvm/src/main/scala/scalacache/logging/Logger.scala

```diff
@@ -1,23 +1,26 @@
 package scalacache.logging
 
 import org.slf4j.{Logger => Slf4jLogger, LoggerFactory}
+import cats.effect.Sync
+import cats.Applicative
+import cats.implicits._
```

Comment:
You don't appear to need this, based on what you are using I'd suggested targeted syntax:

```scala
import cats.syntax.option._
import cats.syntax.flatMap._
```